scripts/create_additive_authoritative_structures.sh

```diff
@@ -150,7 +150,7 @@ function main {
   then
     build_authoritative
   fi
-  echo "$OUTPUT"| sed -e 's/.*/{"data": "&/; s/$/"}/'
+  jq -n --arg output "$OUTPUT" '{data:$output}'
```

Comment:
Is there a way to do this without jq?

Reply:
this could most likely be done with sed.